integration_tests/macros/e2e_tests/tests_validation.sql

```diff
@@ -197,3 +197,7 @@
     {% set result = elementary.run_query(query) %}
     {% do assert_empty_table(result, "dbt_invocations") %}
 {% endmacro %}
+
+{% macro assert_no_dummy_model_table() %}
+    {{ load_relation(ref('dummy_model')) is none }}
```

Comment:
A couple of comments here:

1. I think it will be better to do `{% do return(load_relation(ref('dummy_model')) is none) %}` so the return value will be a boolean rather than a string containing "true".
2. This macro is a bit inconsistent with the other "assert" macros in the sense that it returns a value rather than print "SUCCESS" / "FAILURE" messages, so I think either:
- Remove the `assert_` prefix
- Change this method to print the messages directly